core/scripts/cre/environment/environment/workflow.go

```diff
@@ -53,7 +68,7 @@ func workflowCmds() *cobra.Command {
 	return workflowCmd
 }
 
-func deleteAllWorkflows(ctx context.Context, rpcURL, workflowRegistryAddress string) error {
+func deleteAllWorkflows(ctx context.Context, rpcURL, workflowRegistryAddress, contractsVersion string) error {
```

Comment:
we should just accept a `uint8`.  The string makes it more complicated than necessary IMO.